Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -811,6 +811,22 @@ variable {I J} in
 theorem IsRadical.inf (hI : IsRadical I) (hJ : IsRadical J) : IsRadical (I ⊓ J) := by
   rw [IsRadical, radical_inf]; exact inf_le_inf hI hJ
 
+/-- `Ideal.radical` as an `InfTopHom`, bundling in that it distributes over `inf`. -/
+def radical_hom : InfTopHom (Ideal R) (Ideal R) where
```

Comment:
```suggestion
def radicalInfTopHom : InfTopHom (Ideal R) (Ideal R) where
```
because I think there are multiple ways we might want to bundle this up.

FTR: I tried making this the actual definition of `radical`, but it breaks to much downstream that I don't think it is worth doing it. At least not in this PR.

bors d+